tests/helpers.py

```diff
@@ -28,7 +28,12 @@
 from ocs_ci.ocs.resources.ocs import OCS
 from ocs_ci.ocs.exceptions import CommandFailed, ResourceWrongStatusException
 from ocs_ci.utility.retry import retry
-from ocs_ci.utility.utils import TimeoutSampler, ocsci_log_path, run_cmd
```

Comment:
I would place this somewhere in ocs_ci module, because less code we have in helpers scattered across test code, the better.